includes/discovery/functions.inc.php

```diff
@@ -382,64 +382,39 @@ function discover_sensor(&$valid, $class, $device, $oid, $index, $type, $descr,
 
 function sensor_low_limit($class, $current)
 {
-    $limit = null;
-
     switch ($class) {
         case 'temperature':
             $limit = $current - 10;
             break;
         case 'voltage':
-                $limit = $current * 0.85;
+            $limit = $current * 0.85;
             break;
         case 'humidity':
             $limit = 30;
             break;
-        case 'count':
-        case 'current':
-            $limit = null;
-            break;
         case 'fanspeed':
             $limit = $current * 0.80;
             break;
-        case 'power':
-            $limit = null;
-            break;
-        case 'power_consumed':
```

Comment:
power_consumed and power_factor are both -1 for the limit

Reply:
Updated description of the PR to clarify that this change is intended.